code/game/machinery/ecto_sniffer.dm

```diff
@@ -34,7 +34,7 @@
 /obj/machinery/ecto_sniffer/proc/activate(mob/activator)
 	flick("ecto_sniffer_flick", src)
 	playsound(loc, 'sound/machines/ectoscope_beep.ogg', 25)
-	visible_message("<span class='notice'>[src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!</span>")
+	visible_message("<span class='notice'>\The [src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!</span>")
```

Comment:
This will cause it to say "The The ectoscopic sniffer beeps"

Reply:
Reading this again, it looks like it will actually do nothing as it's cancelled out